lib/main.dart

```diff
@@ -1,6 +1,8 @@
 import 'package:flutter/material.dart';
 import 'package:presentation/my_app.dart';
+import 'di/app_injector.dart';
 
-void main() {
+void main()  async {
```

Comment:
formatting (use pretty printer command + option + L in case of android studio, I'm not sure how it works in vs code)